src/dao/RestDAO.js

```diff
@@ -0,0 +1,17 @@
+// @flow
+
+import Subscription from './Subcription';
+
+class RestDAO extends Subscription {
+  deleteByID() {}
```

Comment:
Are you going to inherit from this? Like `ProductFirmwareDAO extends RestDAO`?

Reply:
yeah, I thought about that, what is the sense of RestDAO if not that? only keeping cache things?